### PR TITLE
Updated cleaning info

### PR DESCRIPTION
**Pull request by ddbb07**
- Updated cleaning info for LotD to v5.5.2
- Updated cleaning info for WACCF to v2.4
- Updated cleaning info for Immersive Encounters to v3.2
- Added cleaning info for Animated Eating Redux for 4.4.0
- Updated cleaning info for Undeath - Classical Lichdom to v2.40
- Added cleaning info for Conditional Expressions for 1.12
- Updated cleaning info for Serana Dialogue Add-On to v2.6.3
- Updated cleaning info for Savage Skyrim to v3.95c
- Updated cleaning info for Dynamic Things Alternative to v0.10.1
- Updated cleaning info for Convenient Carriages to v5.8.0 Main CRC: 1C7037D9
- Updated cleaning info for TPOS-C to v9.2.3

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -3089,13 +3089,23 @@ plugins:
     inc: [ *LotD_v5 ]
 
 ###### AudioVisual - Animations & Physics ######
+  - name: 'AnimatedEatingRedux.esp'
```
Comment (sibir-ine) [inline masterlist.yaml:3092]: @ddbb07 Can you please add a URL for this mod?

File: masterlist.yaml [hunk 2]
```diff
@@ -3089,13 +3089,23 @@ plugins:
...
+        util: 'SSEEdit v4.0.3g'
+
   - name: 'beltfastenedquivers.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/1182/' ]
     msg:
       - <<: *alreadyInX
         subs: [ 'XP32 Maximum Skeleton Special Extended' ]
         condition: 'active("beltfastenedquivers.esp") and active("XPMS(S)?E.esp")'
 
+  - name: 'Conditional Expressions.esp'
```
Comment (sibir-ine) [inline masterlist.yaml:3104]: Can you please add a URL for this mod too?

File: masterlist.yaml [hunk 3]
```diff
@@ -10058,7 +10068,7 @@ plugins:
       - *optional
       - *requiresBashOrSmash
     clean:
-      - crc: 0x1D19D916
+      - crc: 0x7C1ABB8D
```
Comment (sibir-ine) [inline masterlist.yaml:10071]:
```suggestion
      - crc: 0x1D19D916
```

This is the checksum of v2.4 of `Weapons Armor Clothing & Clutter Fixes.esp`, not `WACCF_BashedPatchLvlListFix.esp`.

File: masterlist.yaml [hunk 4]
```diff
@@ -14921,7 +14929,7 @@ plugins:
         subs: [ 'The Great City of Solitude' ]
         condition: 'active("The Great City of Solitude.esp") and not active("CC_TGCS_Patch.esp")'
     clean:
-      - crc: 0xA69C30B9
+      - crc: 0x1C7037D9
```
Comment (sibir-ine) [inline masterlist.yaml:14932]: This is the checksum of `Convenient Carriages.esp` with the **"Beyond Skyrim - Bruma SE" and "Helgen Reborn"** & **Travel speed tweak - Requires SKSE** options selected in the FOMOD installer. The previous checksum was with the **No optional mods installed** & **Travel speed tweak - Requires SKSE** options selected. It seems excessive to include cleaning data for every possible combination, so I think it'd be best to just report it for the default version & maybe the default + SKSE combo.

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by ddbb07** [comment]
Just for your information, this was meant as a draft but i forgot to mark it

**Comment by pStyl3** [comment]
@ddbb07 Hey, would it be possible for you to change the things @sibir-ine mentioned? That way we could then merge this PR. And if you'd like to add further cleaning data, that could be done via additional PRs.

File: masterlist.yaml (hunk 1 above)
Comment (MacSplody) [inline masterlist.yaml:3092]:
```suggestion
  - name: 'AnimatedEatingRedux.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/4652/' ]
```

File: masterlist.yaml (hunk 2 above)
Comment (MacSplody) [inline masterlist.yaml:3104]:
```suggestion
  - name: 'Conditional Expressions.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/45148/' ]
```

**Comment by MacSplody** [comment]
Closing as I'm updating some of these files.

**Pull request closed without merging**